source/mir/cv/core/simd.d

```diff
@@ -0,0 +1,77 @@
+/+
+    Utilities to help use SIMD vectors.
++/
+
+module mir.cv.core.simd;
+
+import cpuid.x86_any;
+import std.traits : isNumeric, TemplateOf;
+
+extern(C) @system nothrow @nogc:
+
+private
+{
+    // TODO: Do we need this?
+    struct Init_cpu_id
+    {
+        static void init()
+        {
+            cpuid_x86_any_init();
+        }
+    }
+    static Init_cpu_id __cpuid_init;
```

Comment:
We need this to get L1/L2/L3 cache levels in the future. In the same time I remember Walter and Andrei proposed `extern(C) shared this()` module constructors that would work in betterC.